src/app/content/search/utils.ts

```diff
@@ -152,7 +158,12 @@ export const highlightResults = (
       return {result: hit, highlights: {}};
     }
 
-    const hitHighlights = hit.highlight.visibleContent.map((highlightText, index) => {
+    // necessary for correct search highlighting of key term results
+    if (isKeyTermHit(hit) && hit.highlight.title) {
```

Comment:
long term i don't think this is the right way to do this, but we can do it for now. if we were to develop different element types that we search each might have different logic for how it becomes highlighted.